c++/src/capnp/compat/json.h

```diff
@@ -48,6 +48,8 @@ class JsonCodec {
   // - When encoding, all primitive fields are always encoded, even if default-valued. Pointer
   //   fields are only encoded if they are non-null.
   // - 64-bit integers are encoded as strings, since JSON "numbers" are double-precision floating
+  // - NaNs and infinite floating point numbers are not allowed by the JSON spec, and so are encoded
```

Comment:
You seem to have inserted this bullet point into the middle of another bullet point. :)